crates/rerun_c/src/rerun.h

```diff
@@ -18,6 +18,33 @@ extern "C" {
 // ----------------------------------------------------------------------------
 // Types:
 
+/// A Utf8 string with a length in bytes.
+typedef struct rr_string {
+    /// Pointer to a UTF8 string.
+    ///
+    /// Does *not* need to be null-terminated.
+    /// Rerun is guaranteed to not read beyond utf8[length_in_bytes-1].
+    const char* utf8;
+
+    /// The length of the string in bytes, *excluding* null-terminator.
```

Comment:
The current wording sort of makes it sounds like there should be a null-terminator after all

```suggestion
    /// The length of the string in bytes (*excluding* null-terminator, if any).
```